concepticondata/conceptlists/Grierson-1928-168.tsv

```diff
@@ -0,0 +1,169 @@
+ID	NUMBER	PAGENUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Grierson-1928-168-1	1	2-3	One	1493	ONE
+Grierson-1928-168-2	2	4-5	Two	1498	TWO
+Grierson-1928-168-3	3	6-7	Three	492	THREE
+Grierson-1928-168-4	4	8-9	Four	1500	FOUR
+Grierson-1928-168-5	5	10-11	Five	493	FIVE
+Grierson-1928-168-6	6	12-13	Six	1703	SIX
+Grierson-1928-168-7	7	14-15	Seven	1704	SEVEN
+Grierson-1928-168-8	8	16-17	Eight	1705	EIGHT
+Grierson-1928-168-9	9	18-19	Nine	1483	NINE
+Grierson-1928-168-10	10	20-21	Ten	1515	TEN
+Grierson-1928-168-11	11	22-23	Twenty	1710	TWENTY
+Grierson-1928-168-12	12	24-25	Fifty	1717	FIFTY
+Grierson-1928-168-13	13	26-27	Hundred	1634	HUNDRED
+Grierson-1928-168-14	14	28-29	I	1209	I
+Grierson-1928-168-15	15	30-31	My	2090	MY
+Grierson-1928-168-16	16	32-33	We	1212	WE
+Grierson-1928-168-17	17	34-35	Our	780	OUR
+Grierson-1928-168-18	18	36-37	Thou	1215	THOU
+Grierson-1928-168-19	19	38-39	Thy	732	YOUR (SINGULAR)
+Grierson-1928-168-20	20	40-41	You	1213	YOU
+Grierson-1928-168-21	21	42-43	Your	2091	YOUR
+Grierson-1928-168-22	22	44-45	He	2642	HE OR SHE
+Grierson-1928-168-23	23	46-47	His	2092	HIS OR HER
+Grierson-1928-168-24	24	48-49	They	817	THEY
+Grierson-1928-168-25	25	50-51	Their	2400	THEIR
+Grierson-1928-168-26	26	52-53	Hand	1277	HAND
+Grierson-1928-168-27	27	54-55	Foot	2098	FOOT OR LEG
```

Comment:
Why not FOOT?

Reply:
Semantic merger between foot and leg in multiple Indian languages.

Reply:
Hm, but not in all? So the word elicited always will refer to foot, but not necessarily to the whole leg? Then I'd map it to FOOT.

Reply:
Okay. Good.